README.md

```diff
@@ -1148,6 +1148,83 @@ admin.createTopics(topics, (err, res) => {
 
 See [createTopics](#createtopicstopics-cb)
 
+### describeConfigs(payload, cb)
+
+Fetch the configuration for the specified resources. It requires Kafka 0.11+.
+
+* `payload`: **Array**, array of resources
+* `cb`: **Function**, the callback
+
+Example:
+
+```js
+const resource = {
+  resourceType: '2',        // '2' for topic, '4' for broker
```

Comment:
should export a constant map instead, and do like resourceType: RESOURCE_TYPES.TOPIC

magic numbers as an api is bad

Reply:
Make sense and i agree, i just did not see any other examples within the library already that forced library consumers to import definitions of a request variable.

"Magic numbers" are already being used in the producer:
```
// Partitioner type (default = 0, random = 1, cyclic = 2, keyed = 3, custom = 4), default 0
    partitionerType: 2
```
which I'm assuming are just the kafka defined values (which I have done the same in this PR).

As an alternative to a new constant map, we could allow users to pass in the string names of the resource type e.g. `broker`, `topic`. I have no preferences.

@aikar @hyperlink thoughts?

Reply:
just because bad practice was used else where, doesn't mean it should continue :)

strings are more readable, but also vulnerable to typos.

but it could be a 
```javascript
const typeMap = {
topic: 2,
broker: 4
}

export const TYPES = {
    TOPIC: "topic",
    BROKER: "broker",
};

const type = typeMap[userOption] || userOption;

```
then the enum map can be provided, but string versions would still work (and even numbers for anyone relying on implementation details)

Reply:
Good suggestions @aikar. 👍 

We can make it a string to number map and yield an error if it's not in the map.

Reply:
Yes I love this approach, thanks @aikar ! I'll push up the changes for this soon.

Reply:
Pushed up the changes for this. I added `RESOURCE_TYPES` directly onto `Admin` so user's don't have to separately import the resource types if that's the only spot that they're going to use it. Also updated the README as I had some pieces wrong, Let me know if there are any changes you'd like to see (any nitpicks as well).

I'll also probably refactor the validation logic in `kafkaClient` soon when I implement the `AlterConfigs` protocol.